src/ErrorHandler/ErrorConverter.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types = 1);
+
+namespace Apitte\Core\ErrorHandler;
```

Comment:
It is closely related with ErrorHandler, but namespace is not ideal. We may change it to `Error` or `ErrorHandling`